packages/pn-personagiuridica-webapp/src/components/Contacts/AddSpecialContactDialog.tsx

```diff
@@ -217,22 +222,23 @@ const AddSpecialContactDialog: React.FC<Props> = ({
                 </MenuItem>
               ))}
           </CustomDropdown>
-          {formik.values.channelType !== ChannelType.SERCQ_SEND && (
-            <TextField
-              size="small"
-              fullWidth
-              id="s_value"
-              name="s_value"
-              label={t(`special-contacts.link-${formik.values.channelType.toLowerCase()}-label`, {
-                ns: 'recapiti',
-              })}
-              value={formik.values.s_value}
-              onChange={handleChangeTouched}
-              error={formik.touched.s_value && Boolean(formik.errors.s_value)}
-              helperText={formik.touched.s_value && formik.errors.s_value}
-              sx={{ mb: 2 }}
-            />
-          )}
+          {formik.values.channelType !== ChannelType.SERCQ_SEND &&
```

Comment:
change the formik.values.channelType !== ChannelType.SERCQ_SEND && formik.values.channelType !== '' with formik.values.channelType === ChannelType.PEC